lib/bambora/bambora/service/submit_single_payment.ex

```diff
@@ -73,12 +93,14 @@ defmodule Bambora.Service.SubmitSinglePayment do
         account_number: account_number
       }) do
     [
+      X.element("CustomerStorageNumber", "VAULT1"),
```

Comment:
@dagoof I guess this is a magic value?

Reply:
seems like it. not documented anywhere but just in their examples

https://dev-apac.bambora.com/checkout/guides/customer-registration/managing-cards

we probably need to get in contact and ask